src/utils.rs

```diff
@@ -170,8 +170,13 @@ where
 {
     use serde::Deserialize;
     let s = String::deserialize(deserializer)?;
-    let wkt = wkt::Wkt::from_str(&s).map_err(serde::de::Error::custom)?;
-    try_into_geometry(&wkt.items[0]).map_err(serde::de::Error::custom)
+    let mut wkt = wkt::Wkt::from_str(&s).map_err(serde::de::Error::custom)?;
+    use std::convert::TryInto;
+    wkt.items
+        .pop()
+        .unwrap()
+        .try_into()
```

Comment:
We now utilize idiomatic Rust's `TryInto` trait, rather than having our own function only for wkt.

Reply:
You should be able to replace it with this

```rust
let wkt = wkt::Wkt::from_str(&s).map_err(serde::de::Error::custom)?;
    use std::convert::TryInto;
    wkt.try_into().map_err(serde::de::Error::custom)
```

Reply:
indeed! Fixed.

Do you think the WKT crate should add methods like these for optional `serde` integration? It seems likely to be useful to other people.

Reply:
It actually might be helpful indeed, so I took the opportunity to contribute back. See https://github.com/georust/wkt/pull/59.

Reply:
Wonderful! I'll take a look.